recipes/pillow-jpls/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "pillow-jpls" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pillow_jpls-{{ version }}.tar.gz
+  sha256: 9cb643dd50c8fa848c16fe5e41350049ca762499e22cf5fb4cd5f67ae2c027d1
+
+build:
+  script: |
+    {% if win %}
+    set "CMAKE_ARGS=%CMAKE_ARGS% -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=%PREFIX%\Library"
+    if "%CMAKE_GENERATOR%"=="Ninja" (
+      set "CMAKE_ARGS=%CMAKE_ARGS% -DCMAKE_C_COMPILER=cl.exe -DCMAKE_CXX_COMPILER=cl.exe"
+    )
+    {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --no-clean
+    {% else %}
+    {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --no-clean
+    {% endif %}
```

```suggestion
    {{ PYTHON }} -m pip -C cmake.args="--preset sysdeps" install . -vv --no-deps --no-build-isolation --no-clean
    {% else %}
    {{ PYTHON }} -m pip -C cmake.args="--preset sysdeps" install . -vv --no-deps --no-build-isolation --no-clean
    {% endif %}
```
Please make this change to the build call so that the build process does not use vendored dependencies.  This is causing other issues with the build and needs to be disabled.